src/styles/constants.js

```diff
@@ -23,3 +23,13 @@ export const shadows = {
   DOUBLE: 'double',
   TRIPLE: 'triple'
 };
+
+/**
+ * Colors?
+ */
+
+export const colorNames = {
```

Comment:
I think pre-defining semantic colors makes a lot of sense. And if we use them for modifier styles, having constants like this is also good. But I also thing we should add these colors to the theme (`theme.colors.warning`). I think there already is a section for that.

Reply:
Perfect, I was also thinking that but wondering whether it's hard to define globally when you might have different contrasts for different semantic colors. Will go ahead and add 'em.